src/.vuepress/components/common/codepen-snippet.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <p
+    class="codepen"
+    data-theme-id="39028"
```

Comment:
Some of the current examples use a different value here.

I'm not sure what it does but is the change intentional?

Reply:
I really think its a problem, the standardized theme should be used along all the docs. But I don't want to change behavior with this PR, so I've created a new `theme` prop too. And it'll be easy to solve the different theme in some files in the future, if the team wants.